### PR TITLE
feat: refactoring event page with new styling, additional info box an…

### PR DESCRIPTION
**Pull request by jxmoose**
TODO: need to add emails for group sizing + post confirmation styling

[//]: # "Feel free to customize this template to your project's vibes"

### :croissant: Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

### :baguette_bread: Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

[image placeholder]: 

## :sandwich: How to review
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'

## :pie: Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

### :pretzel: Online sources
[//]: # 'Copy links to any tutorials or documentation that was useful to you when working on this PR'

### :bagel: Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @jxmoose

### CODE REVIEW

File: app/discover/[event_id]/page.tsx [hunk 1]
```diff
@@ -104,29 +189,32 @@ export default function EventPage({
             <IndividualTag $bgColor={COLORS.bread6}>
               {event?.performance_type}
             </IndividualTag>
-            <IndividualTag $bgColor={COLORS.lilac3}>
-              {event?.genre}
-            </IndividualTag>
+            {event?.genre && (
+              <IndividualTag $bgColor={COLORS.lilac3}>
```
Comment (allisonhongberkeley) [inline app/discover/[event_id]/page.tsx:193]: in v6, the tags are above the text?

**Comment by allisonhongberkeley** [comment]
lgtm 🚀

**Review by allisonhongberkeley — APPROVED** [review]
- small nit but the location icon and location text under the facility name doesn't look centered 
Screenshot 2025-03-07 at 4 17 14 PM

o/w lgtm! 🚀
